ddcommon-ffi/cbindgen.toml

```diff
@@ -22,12 +22,23 @@ after_includes = """
 /* NOTE: Compilation fails if you pass in a char* instead of a literal */ ((ddog_CharSlice){ .ptr = "" string, .len = sizeof(string) - 1 })
 #endif
 
+#if defined __GNUC__
+#  define DDOG_GNUC_VERSION(major) __GNUC__ >= 4
```

Comment:
`major` never gets used -- was the intention to use `__GNUC__ >= major` in the macro? 🤔

Reply:
oh yeah, good one. Will fix it

Reply:
done in https://github.com/DataDog/libdatadog/pull/48/commits/114ba3cf0115eee04577be424f693abed58d0956